### ISSUE TITLE
nnet training failing on chime6

### ISSUE THREAD

**Issue by aarora8** [espnet/espnet#3647]
**Describe the bug**
I am running CHiME-6 egg 1 script. At stage 6 in run.sh (https://github.com/espnet/espnet/blob/master/egs/chime6/asr1/run.sh). I am getting the following error "unrecognized arguments: --atype=location --adim=256 --aheads=1 --awin=5 --aconv-chans=10 --aconv-filts=100 --rnnt-mode=rnnt". I did not made any modification in the scripts.  

**Basic environments:**
 - OS information: "CentOS Linux release 7.5.1804 (Core)" (Linux 3.10.0-862.11.6.el7.x86_64 #1 SMP Tue Aug 14 21:49:04 UTC 2018 x86_64)
 - python version: [3.8.11 (default, Aug  3 2021, 15:09:35)  [GCC 7.5.0]]
 - espnet version: [0.10.3a4]
 - pytorch version [torch=1.8.1]
 - git hash 66facd2859977d23349cf74d5c8f5394e5cb6784
 - Commit date: `Sun Oct 3 12:14:15 2021 +0900`
 
Versions of relevant libraries:
[pip3] numpy==1.20.3
[pip3] pytorch-ranger==0.1.1
[pip3] pytorch-wpe==0.0.1
[pip3] torch==1.8.1
[pip3] torch-complex==0.2.1
[pip3] torch-optimizer==0.1.0
[pip3] torch-tb-profiler==0.2.1
[pip3] torchaudio==0.8.1
[conda] blas                      1.0                         mkl
[conda] cudatoolkit               10.2.89              hfd86e86_1
[conda] mkl                       2021.3.0           h06a4308_520
[conda] mkl-service               2.4.0            py38h7f8727e_0
[conda] mkl_fft                   1.3.0            py38h42c9631_2
[conda] mkl_random                1.2.2            py38h51133e4_0
[conda] numpy                     1.20.3           py38hf144106_0
[conda] numpy-base                1.20.3           py38h74d4b33_0
[conda] pytorch                   1.8.1           py3.8_cuda10.2_cudnn7.6.5_0    pytorch
[conda] pytorch-ranger            0.1.1                    pypi_0    pypi
[conda] pytorch-wpe               0.0.1                    pypi_0    pypi
[conda] torch-complex             0.2.1                    pypi_0    pypi
[conda] torch-optimizer           0.1.0                    pypi_0    pypi
[conda] torchaudio                0.8.1                    pypi_0    pypi
```
You can obtain them by the following command

```
cd <espnet-root>/tools
. ./activate_python.sh
python3 -m torch.utils.collect_env
```

**Task information:**
 - Task: [e.g., ASR]
 - Recipe: [e.g. chime6]
 - ESPnet1 or ESPnet2

**To Reproduce**
Steps to reproduce the behavior:
1. move to a recipe directory, e.g., `cd egs/chime6/asr1/`
2. execute `run.sh` with specific arguments, e.g., `run.sh --stage 6 --ngp 1`
3. specify the error log, e.g., `exp/train_worn_simu_u400k_cleaned_trim_sp_pytorch_train/train.log`

**Error logs**
Paste the error logs. If applicable, add screenshots to help explain your problem.
# asr_train.py --config conf/train.yaml --ngpu 3 --backend pytorch --outdir exp/train_worn_simu_u400k_cleaned_trim_sp_pytorch_train/results --tensorboard-dir tensorboard/train_worn_simu_u400k_cleaned_trim_sp_pytorch_train --debugmode 1 --dict data/lang_1char/train_worn_simu_u400k_cleaned_trim_sp_units.txt --debugdir exp/train_worn_simu_u400k_cleaned_trim_sp_pytorch_train --minibatches 0 --verbose 0 --resume --train-json dump/train_worn_simu_u400k_cleaned_trim_sp/deltafalse/data.json --valid-json dump/dev_gss_u1k/deltafalse/data.json --preprocess-conf conf/specaug.yaml
# Started at Sun Oct 10 11:14:52 EDT 2021
#
usage: asr_train.py [-h] [--config CONFIG] [--config2 CONFIG2]
                    [--config3 CONFIG3] [--ngpu NGPU]
                    [--train-dtype {float16,float32,float64,O0,O1,O2,O3}]
                    [--backend {chainer,pytorch}] --outdir OUTDIR
                    [--debugmode DEBUGMODE] --dict DICT [--seed SEED]
                    [--debugdir DEBUGDIR] [--resume [RESUME]]
                    [--minibatches MINIBATCHES] [--verbose VERBOSE]
                    [--tensorboard-dir [TENSORBOARD_DIR]]
                    [--report-interval-iters REPORT_INTERVAL_ITERS]
                    [--save-interval-iters SAVE_INTERVAL_ITERS]
                    [--train-json TRAIN_JSON] [--valid-json VALID_JSON]
                    [--model-module MODEL_MODULE] [--num-encs NUM_ENCS]
                    [--ctc_type {builtin,warpctc,gtnctc,cudnnctc}]
                    [--mtlalpha MTLALPHA] [--lsm-weight LSM_WEIGHT]
                    [--report-cer] [--report-wer] [--nbest NBEST]
                    [--beam-size BEAM_SIZE] [--penalty PENALTY]
                    [--maxlenratio MAXLENRATIO] [--minlenratio MINLENRATIO]
                    [--ctc-weight CTC_WEIGHT] [--rnnlm RNNLM]
                    [--rnnlm-conf RNNLM_CONF] [--lm-weight LM_WEIGHT]
                    [--sym-space SYM_SPACE] [--sym-blank SYM_BLANK]
                    [--sortagrad [SORTAGRAD]]
                    [--batch-count {auto,seq,bin,frame}]
                    [--batch-size BATCH_SIZE] [--batch-bins BATCH_BINS]
                    [--batch-frames-in BATCH_FRAMES_IN]
                    [--batch-frames-out BATCH_FRAMES_OUT]
                    [--batch-frames-inout BATCH_FRAMES_INOUT] [--maxlen-in ML]
                    [--maxlen-out ML] [--n-iter-processes N_ITER_PROCESSES]
                    [--preprocess-conf [PREPROCESS_CONF]]
                    [--opt {adadelta,adam,noam}] [--accum-grad ACCUM_GRAD]
                    [--eps EPS] [--eps-decay EPS_DECAY]
                    [--weight-decay WEIGHT_DECAY]
                    [--criterion {loss,loss_eps_decay_only,acc}]
                    [--threshold THRESHOLD] [--epochs EPOCHS]
                    [--early-stop-criterion [EARLY_STOP_CRITERION]]
                    [--patience [PATIENCE]] [--grad-clip GRAD_CLIP]
                    [--num-save-attention NUM_SAVE_ATTENTION]
                    [--num-save-ctc NUM_SAVE_CTC] [--grad-noise GRAD_NOISE]
                    [--num-spkrs {1,2}]
                    [--context-residual [CONTEXT_RESIDUAL]]
                    [--enc-init ENC_INIT] [--enc-init-mods ENC_INIT_MODS]
                    [--dec-init DEC_INIT] [--dec-init-mods DEC_INIT_MODS]
                    [--freeze-mods FREEZE_MODS] [--use-frontend USE_FRONTEND]
                    [--use-wpe USE_WPE]
                    [--wtype {lstm,blstm,lstmp,blstmp,vgglstmp,vggblstmp,vgglstm,vggblstm,gru,bgru,grup,bgrup,vgggrup,vggbgrup,vgggru,vggbgru}]
                    [--wlayers WLAYERS] [--wunits WUNITS] [--wprojs WPROJS]
                    [--wdropout-rate WDROPOUT_RATE] [--wpe-taps WPE_TAPS]
                    [--wpe-delay WPE_DELAY]
                    [--use-dnn-mask-for-wpe USE_DNN_MASK_FOR_WPE]
                    [--use-beamformer USE_BEAMFORMER]
                    [--btype {lstm,blstm,lstmp,blstmp,vgglstmp,vggblstmp,vgglstm,vggblstm,gru,bgru,grup,bgrup,vgggrup,vggbgrup,vgggru,vggbgru}]
                    [--blayers BLAYERS] [--bunits BUNITS] [--bprojs BPROJS]
                    [--badim BADIM] [--bnmask BNMASK]
                    [--ref-channel REF_CHANNEL]
                    [--bdropout-rate BDROPOUT_RATE] [--stats-file STATS_FILE]
                    [--apply-uttmvn APPLY_UTTMVN]
                    [--uttmvn-norm-means UTTMVN_NORM_MEANS]
                    [--uttmvn-norm-vars UTTMVN_NORM_VARS]
                    [--fbank-fs FBANK_FS] [--n-mels N_MELS]
                    [--fbank-fmin FBANK_FMIN] [--fbank-fmax FBANK_FMAX]
                    [--etype {custom,lstm,blstm,lstmp,blstmp,vgglstmp,vggblstmp,vgglstm,vggblstm,gru,bgru,grup,bgrup,vgggrup,vggbgrup,vgggru,vggbgru}]
                    [--dropout-rate DROPOUT_RATE] [--elayers ELAYERS]
                    [--eunits EUNITS] [--eprojs EPROJS]
                    [--subsample SUBSAMPLE] [--enc-block-arch ENC_BLOCK_ARCH]
                    [--enc-block-repeat ENC_BLOCK_REPEAT]
                    [--custom-enc-input-layer {conv2d,vgg2l,linear,embed}]
                    [--custom-enc-positional-encoding-type {abs_pos,scaled_abs_pos,rel_pos}]
                    [--custom-enc-self-attn-type {self_attn,rel_self_attn}]
                    [--custom-enc-pw-activation-type {relu,hardtanh,selu,swish}]
                    [--custom-enc-conv-mod-activation-type {relu,hardtanh,selu,swish}]
                    [--dtype {lstm,gru,custom}]
                    [--dropout-rate-decoder DROPOUT_RATE_DECODER]
                    [--dropout-rate-embed-decoder DROPOUT_RATE_EMBED_DECODER]
                    [--dec-embed-dim DEC_EMBED_DIM] [--dlayers DLAYERS]
                    [--dunits DUNITS] [--dec-block-arch DEC_BLOCK_ARCH]
                    [--dec-block-repeat DEC_BLOCK_REPEAT]
                    [--custom-dec-input-layer {linear,embed}]
                    [--custom-dec-pw-activation-type {relu,hardtanh,selu,swish}]
                    [--optimizer-warmup-steps OPTIMIZER_WARMUP_STEPS]
                    [--noam-lr NOAM_LR] [--noam-adim NOAM_ADIM]
                    [--transformer-warmup-steps OPTIMIZER_WARMUP_STEPS]
                    [--transformer-lr NOAM_LR] [--adim NOAM_ADIM]
                    [--transducer-weight TRANSDUCER_WEIGHT]
                    [--joint-dim JOINT_DIM]
                    [--joint-activation-type {relu,tanh,swish}]
                    [--score-norm [SCORE_NORM]]
                    [--fastemit-lambda FASTEMIT_LAMBDA]
                    [--use-ctc-loss [USE_CTC_LOSS]]
                    [--ctc-loss-weight CTC_LOSS_WEIGHT]
                    [--ctc-loss-dropout-rate CTC_LOSS_DROPOUT_RATE]
                    [--use-lm-loss [USE_LM_LOSS]]
                    [--lm-loss-weight LM_LOSS_WEIGHT]
                    [--lm-loss-smoothing-rate LM_LOSS_SMOOTHING_RATE]
                    [--use-aux-transducer-loss [USE_AUX_TRANSDUCER_LOSS]]
                    [--aux-transducer-loss-weight AUX_TRANSDUCER_LOSS_WEIGHT]
                    [--aux-transducer-loss-enc-output-layers AUX_TRANSDUCER_LOSS_ENC_OUTPUT_LAYERS]
                    [--aux-transducer-loss-mlp-dim AUX_TRANSDUCER_LOSS_MLP_DIM]
                    [--aux-transducer-loss-mlp-dropout-rate AUX_TRANSDUCER_LOSS_MLP_DROPOUT_RATE]
                    [--use-symm-kl-div-loss [USE_SYMM_KL_DIV_LOSS]]
                    [--symm-kl-div-loss-weight SYMM_KL_DIV_LOSS_WEIGHT]
asr_train.py: error: unrecognized arguments: --atype=location --aheads=1 --awin=5 --aconv-chans=10 --aconv-filts=100 --rnnt-mode=rnnt
# Accounting: time=5 threads=1
# Ended (code 2) at Sun Oct 10 11:14:57 EDT 2021, elapsed time 5 seconds
